pyopia/pipeline.py

```diff
@@ -82,11 +104,18 @@ def run(self):
             stats (DataFrame): stats DataFrame of particle statistics
         '''
 
-        timestamp, imraw = self.steps['load']()
-        imc = self.steps['imageprep'](imraw, self.common)
-        stats = self.steps['statextract'](timestamp, imc, self.cl)
+        self.common['steps_string'] = steps_to_string(self.common['steps'])
+
+        blacklist_steps = ['common', 'classifier']
+        # [self.steps[s]() for s in self.steps if not start.__contains__(s)]
+        for s in self.common['steps']:
+            if blacklist_steps.__contains__(s):
```

Comment:
is:
if s in blacklist_steps:
    ....
    
The same code functionally? If so that much easier for me to understand.

Reply:
no - this will loop through steps, but skip the steps with blacklisted keys. there might be a more elegant solution here.